src/collectd.conf.pod

```diff
@@ -9,6 +9,7 @@ collectd.conf - Configuration for the system statistics collection daemon B<coll
   BaseDir "/var/lib/collectd"
   PIDFile "/run/collectd.pid"
   Interval 10.0
+  StartRead -1.0
```

Comment:
AlignRead?

Reply:
Ups, I missed to rename this one. What about removing the line? This would have the same effect, unaligned reads as we currently have.

Reply:
My preference would be to leave it there, with a comment on what it does. 

Having a parameter named AlignRead and a default value of -1.0 is not very obvious why and what the result would be.

Reply:
I renamed it and added a comment, which also includes what 0.0 means. Furthermore, I added a sentence on the default value in the description of AlignRead.